documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -120,7 +120,7 @@
                      "counter":0,
                      "roughness":0.0
                   },
-                  "grv_latency_statistics":{
+                  "grv_latency_statistics":{ // GRV Latency metrics are grouped according to priority (currently batch or default).
```

Comment:
I'm not sure what the implications of this comment are, as this file is used to generate documentation. Probably you can remove this one for now, but if you've generated the documentation and think the result looks good, we can keep it.

Reply:
I felt ok adding this as there are other comments in similar positions in this file. After building the docs, the generated html output seemed  a bit more readable when loaded in my browser. It make sense to me to keep it.